content/applications/inventory_and_mrp/inventory/management/products/reordering_rules.rst

```diff
@@ -0,0 +1,126 @@
+- :guilabel:`Multiple Quantity`: specify if the product should be replenished in batches of a
+  certain quantity; for example, a product could be replenished in batches of 20
+- :guilabel:`UoM`: the unit of measure used for reordering the product; this value can simply be
+  `Units` or a specific unit of measurement for weight, length, etc.
+
+.. image:: reordering_rules/reordering-rule-form.png
+   :align: center
+   :alt: The form for creating a new reordering rule.
+
+.. tip::
+   Reordering rules can also be created from each product page. To do so, navigate to
+   :menuselection:`Inventory --> Products --> Products`, then select a product. Click on
+   :menuselection:`Reordering Rules --> Create`, then fill out the new line as detailed above.
+
+Once a reordering rule has been created for a product, if its forecasted quantity is below the
+rule's *Min Quantity* when the scheduler runs, a replenishment order for the product will be
+automatically generated. By default, the scheduler runs once each day.
+
+.. tip::
+   To manually trigger a reordering rule before the scheduler runs, select :menuselection:`Inventory
+   --> Operations --> Run Scheduler`. Then, select the green :guilabel:`Run Scheduler` button on the
+   pop-up that appears. Be aware that this will also trigger any other scheduled actions.
+
+If the :guilabel:`Buy` route is selected, then an :abbr:`RfQ (Request for Quotation)` will be
+generated. To view and manage :abbr:`RFQs (Requests for Quotation)`, navigate to
+:menuselection:`Purchase --> Orders --> Requests for Quotation`.
+
+If the :guilabel:`Manufacture` route is selected, then an :abbr:`MO (Manufacturing Order)` will be
+generated. To view and manage :abbr:`MOs (Manufacturing Orders)`, navigate to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`.
+
+Select a preferred route to use for reordering
+==============================================
+
+Odoo allows for multiple routes to be selected under the :guilabel:`Inventory` tab on each product
+page. For instance, it is possible to select both :guilabel:`Buy` and :guilabel:`Manufacture`, thus
+enabling the functionality of both routes.
+
+Odoo also enables users to set a preferred route for a product's reordering rule. This is the route
+that the rule will default to if multiple are selected. To select a preferred route, begin by
+navigating to :menuselection:`Inventory --> Configuration --> Reordering Rules`.
+
+By default, the :guilabel:`Preferred Route` column is hidden on the :guilabel:`Reordering Rules`
+page. Enable it by selecting the three-dot :guilabel:`Options` button on the right side of the page
+and checking the :guilabel:`Preferred Route` checkbox.
+
+The :guilabel:`Preferred Route` column will now appear. Click inside of the column on the row of a
+reordering rule and a :guilabel:`drop-down` will show all available routes for that rule. Select
+one, and it will be set as the preferred route.
```

Comment:
```suggestion
and checking the :guilabel:`Preferred Route` checkbox. Doing so reveals the :guilabel:`Preferred
Route` column.

Click inside of the column on the row of a reordering rule, and a drop-down menu shows all available
routes for that rule. Select one, and it will be set as the preferred route.
```